nbodykit/base/particles.py

```diff
@@ -119,9 +148,8 @@ def attrs(self):
         except AttributeError:
             self._attrs = {}
             return self._attrs
-
-    @staticmethod
-    def compute(*args, **kwargs):
+            
+    def compute(self, *args, **kwargs):
         """
         Our version of :func:`dask.compute` that computes
         multiple delayed dask collections at once
```

Comment:
1. We don't know when not to issue a warning. Let's keep it as it is then.
2. Sure. tuples of length one is inconsistent with *args.